Atualizando o link para as VMs

leandroramos opened the pull request.

leandroramos committed:
Atualizando o link para as VMs

leandroramos committed:
Merge branch 'master' into update-vm-link

girol commented:
@leandroramos essa PR ainda é válida?

thiagogomesverissimo reviewed (changes requested):
Esse link não tem nada: https://drive.google.com/drive/u/0/folders/1QdLAUp2svOjrjEv3KxWX3WPR8iAmtBbc

leandroramos commented:
Vou corrigir esse link amanhã. Obrigado.

Em Sat, 21 Jul 2018 13:11:09 -0700
Thiago Gomes Verissimo <notifications@github.com> escreveu:

> thiagogomesverissimo requested changes on this pull request.
> 
> Esse link não tem nada:
> https://drive.google.com/drive/u/0/folders/1QdLAUp2svOjrjEv3KxWX3WPR8iAmtBbc
> 
> 
>

leandroramos committed:
Link corrigido

Corrigi o link. Eu havia colocado um link para o nosso drive de equipa, mas ele não permite compartilhar pastas, somente os arquivos com link direto. Quando a pessoa acessa o link direto para o arquivo, já vem o aviso do Google de que não consegue escanear por vírus e pode assustar a pessoa. Acessando a pasta as pessoas podem ver do que se trata antes de receber o alerta, por isso deixei o link no meu próprio drive.

girol committed:
Merge branch 'master' into update-vm-link
